### DISCUSSION TITLE
Error: EISDIR: illegal operation on a directory, read

### DISCUSSION THREAD

**Discussion by flashlog** [jellydn/next-swagger-doc#451] [Q&A]
### Describe the bug

Using the code from the ReadME throws the following error. Tried with cli as well, getting the same error. 
**Operating System: Mac**

![image](https://user-images.githubusercontent.com/64234223/188348569-a18275b9-452e-4df4-94df-a34b450d21cc.png)

### Reproduction

yarn dev

### System Info

```Shell
NON
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by jellydn** [comment]
Hi there, thank you for your report. Could you please share with me reproduce repository? Sorry, I can't reproduce as your report.
